ibm/mas_devops/roles/gencfg_jdbc/templates/jdbccfg.yml.j2

```diff
@@ -0,0 +1,28 @@
+---
+apiVersion: v1
+kind: Secret
+type: Opaque
+metadata:
+  name: "jdbc-{{ db_instance_id | lower }}-credentials"
+  namespace: "mas-{{ mas_instance_id }}-core"
+stringData:
+  username: "{{db_username}}"
+  password: "{{jdbc_instance_password}}"
+---
+apiVersion: config.mas.ibm.com/v1
+kind: JdbcCfg
+metadata:
+  name: "{{ suite_jdbccfg_name }}"
+  namespace: "mas-{{ mas_instance_id }}-core"
+  labels: {{ suite_jdbccfg_labels }}
+spec:
+  displayName: "{{ suite_jdbccfg_name }}-manage"
```

Comment:
Why does the display name have "-manage" at the end? You could call this role to setup jdbccfg for a different app but you will get `-manage` at the end even if it is for monitor?

Reply:
removed the manage tag